app/views/shared/_what_you_will_learn.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="jumbotron">
+  <h3 class="text-center">What You Will Learn</h3>
+  <p class="text-center">Professional developers technologies</p>
+
+  <div class="row text-center">
+    <div class="col-lg-4">
+      <img class="img-circle" src="https://www.w3.org/html/logo/downloads/HTML5_Logo_512.png" alt="HTML logo" width="140" height="140">
```

Comment:
usualy you don't deeplink to other server's resources. You never know what happens to them